src/main/resources/Help/Code/Translations.xml

```diff
@@ -0,0 +1,251 @@
+help.applications.forum.title=Forum
+help.applications.ideas.decription=Ideas Application encourage you to submit ideas that can be voted by your team.
+help.applications.ideas.title=Ideas
+help.applications.meetings.description=Meetings Application manages your meetings mentioning the date, location, participants, etc.
+help.applications.meetings.title=Meetings
+help.applications.menu.description=Menu Application helps you create navigation menus and displayed them in your instance.
+help.applications.menu.title=Menu
+help.applications.movie.title=Movies
+help.applications.title=Applications
+help.applications.viewMoreApplications=View more applications
+help.browse.label=Browse
+help.contentOrganization.description=Organize your content in hierarchical pages, grouped in several (sub)wikis.
+help.contentOrganization.title=Content Organization
+help.gettingStarted.description=Easy steps to get you started with XWiki.
+help.gettingStarted.title=Getting Started
+help.howToUseXwiki.description=XWiki offers a wide range of powerful features that will enable you to create, organize and find content. This section will show you how to use the basic features of XWiki.
+help.history.checkTheHistoryTab=Check the {0}history{0} tab.
+help.history.compareTheVersions=Compare the versions.
+help.history.title=History
+help.howToUseXwiki.title=How to use XWiki?
+help.importExport.description=Import conteknt from various sources and export it in an open format (XML, HTML, PDF, ODT).
+help.importExport.title=Import / Export
+help.learnMore.label=Learn more
+help.macros.barChart=Bar Chart
+help.macros.box.description=Displays a message in a styled box.
+help.macros.box.title=Box
+help.macros.boxMacro.title=Box Macros
+help.macros.boxMacros.description={0}Box Macro{1} displays a message in a styled box.
+help.macros.browseMacros.description=Choose from community contributed macros and install them inside your wiki.
+help.macros.browseMacros.tip=The {0}Macro Index{1} lists the macros that are already installed and ready to be used inside your pages.
+help.macros.browseMacros.title=Browse Macros
+help.macros.chart={0}Chart Macro{1} displays graphical charts generated from miscellaneous data sources.
+help.macros.chart.description=Generates graphical charts from data sources.
+help.macros.chart.title=Chart
+help.macros.description=Macros are reusable functionality that can be integrated inside a page.
+help.macros.errorMacro=Error Macro
+help.macros.examples.description=Need help or confused with some of our macros? Check out their documentation and find out more about their functionality.
+help.macros.example=Example
+help.macros.examples.title=Examples
+help.macros.gallery={0}Gallery Macro{1} displays a list of images in a slide-show gallery.
+help.macros.gallery.description=Displays a list of images in a slide-show gallery.
+help.macros.gallery.title=Gallery
+help.macros.infoMacro=Info Macro
+help.macros.thisIsAnInfoMacro=This is an info message
```

Comment:
Not a big fan of these compound camel keys (.thisIsAnInfoMacro) and I've seen you've used them in several places.